fastlane_core.gemspec

```diff
@@ -28,6 +28,7 @@ Gem::Specification.new do |spec|
   spec.add_dependency 'babosa' # transliterate strings
   spec.add_dependency 'excon', '~> 0.45.0' # Great HTTP Client
   spec.add_dependency 'rubyzip', '~> 1.1.6' # needed for extracting the ipa file
+  spec.add_dependency 'plist', '~> 3.1.0' # needed for parsing provisioning profiles
```

Comment:
Thanks, could you change it to `3.1`? I updated the other `plist` dependency in the other gems too